examples/kotlin-matter-controller/java/src/com/matter/controller/commands/pairing/PairingCommand.kt

```diff
@@ -177,10 +177,10 @@ abstract class PairingCommand(
     logger.log(Level.INFO, "onICDRegistrationInfoRequired")
   }
 
-  override fun onICDRegistrationComplete(icdNodeId: Long, icdCounter: Long) {
+  override fun onICDRegistrationComplete(icdNodeId: Long, icdCounter: Long, symmetricKey: ByteArray) {
```

Comment:
should we also pass user_active_mode_trigger_hint and instruction and monitor subject via this callback? 
maybe we create a class to cover these meta data in case we change it in the future?

Reply:
I modified to class parameter.

Reply:
Followup: please rename ICDDeviceInfo to ICDRegistrationInfo, and put nodeId, counter, monitor subject inside

Reply:
The reason for creating a separate class is because there is a need to later pass UserActiveModeTriggerHint and UserActiveModeTriggerInstruction to the Upper Layer. Of course, according to your opinion, icdCounter, icdNodeId, and MonitoredSubject should also be included, but I think it is inappropriate to include UserActiveModeTriggerHint and UserActiveModeTriggerInstruction in ICDRegistrationInfo.

(Other parameters except symmetric key will be implemented after this PR is merged.)

Reply:
agree, thanks